packages/patterns-react/src/patterns/CardSection/__stories__/CardSection.stories.js

```diff
@@ -77,7 +77,7 @@ const cardsGroup = {
     },
     {
       image: {
-        defaultImage: 'http://picsum.photos/id/1003/1056/480',
+        defaultImage: 'https://picsum.photos/id/1018/1056/480',
```

Comment:
I can't add a comment, but on line `L#67` above. The image isn't rendering. I added `https`, and it seemed to work for me?

Reply:
yeah @photodow: the `s` is missing from the image URL  because of the feature link  commit from @carthick 
![image](https://user-images.githubusercontent.com/53789187/72636739-b8811400-392d-11ea-8f4a-572d7929116a.png)
Let me update the PR.